Week1/homework/js-exercises/ex4.js

```diff
@@ -0,0 +1,10 @@
+//Exercise 4: Log the string
+//Declare a variable
+var myString = 'Dima' + ' ' + 'Kaddah';
```

Comment:
this exercise i did't understand the instructions, but i will use let after i read it carefully because they ask in  step 4 to reassign myString. i really don't understand what i must do. i will try to use const and do another way i think you have idea you want me to know about. so i will make two ways.one with let and one with const.